explore/src/main/scala/explore/config/SpectroscopyModesTable.scala

```diff
@@ -294,7 +294,10 @@ private object SpectroscopyModesTable:
       column(AvailablityColumnId, row => row.rowToConf(cw))
         .setCell(_.value.fold("No")(_ => "Yes"))
         .setColumnSize(FixedSize(66.toPx))
-        .sortable
+        .sortableBy(
+          // None are first, just like in Ordering[Option[?]]
+          _.fold("")(_.configuration.configurationSummary)
```

Comment:
Can we just use `sortableBy(_.map(_.configuration.configurationSummary))` and avoid the `fold`?

Reply:
Yeah. Seems like that should work. I'll try.